cmd/gaia/cli_test/cli_test.go

```diff
@@ -358,26 +358,26 @@ func TestGaiaCLISendGenerateAndSign(t *testing.T) {
 	require.Equal(t, len(msg.Msgs), 1)
 	require.Equal(t, 0, len(msg.GetSignatures()))
 
-	// Test generate sendTx, estimate gas
+	// Test generate sendTx with --gas=$amount
 	success, stdout, stderr = executeWriteRetStdStreams(t, fmt.Sprintf(
-		"gaiacli send %v --amount=10steak --to=%s --from=foo --gas=0 --generate-only",
+		"gaiacli send %v --amount=10steak --to=%s --from=foo --gas=100 --generate-only",
 		flags, barAddr), []string{}...)
 	require.True(t, success)
-	require.NotEmpty(t, stderr)
+	require.Empty(t, stderr)
 	msg = unmarshalStdTx(t, stdout)
-	require.NotZero(t, msg.Fee.Gas)
+	require.Equal(t, msg.Fee.Gas, int64(100))
 	require.Equal(t, len(msg.Msgs), 1)
+	require.Equal(t, 0, len(msg.GetSignatures()))
 
-	// Test generate sendTx with --gas=$amount
+	// Test generate sendTx, estimate gas
 	success, stdout, stderr = executeWriteRetStdStreams(t, fmt.Sprintf(
-		"gaiacli send %v --amount=10steak --to=%s --from=foo --gas=100 --generate-only",
+		"gaiacli send %v --amount=10steak --to=%s --from=foo --gas=0 --generate-only",
 		flags, barAddr), []string{}...)
 	require.True(t, success)
-	require.Empty(t, stderr)
+	require.NotEmpty(t, stderr)
```

Comment:
Can you change this error to test the actual output ? (eg: parsing and using `require.Equal`)

Reply:
#2266 needs to be merged first

Reply:
#2266 has now been merged